tools/summonerd/src/main.rs

```diff
@@ -206,11 +207,13 @@ impl Opt {
                 };
                 let service =
                     CoordinatorService::new(knower, storage.clone(), queue.clone(), marker);
-                let grpc_server = Server::builder().add_service(
+
+                let routes = Routes::new(
```

Comment:
The trusty tonic Server can no longer be turned into an (axum) router - instead, tonic routes need to be constructed.